web-registry/src/components/molecules/MetaDetail/MetaDetail.types.ts

```diff
@@ -0,0 +1,27 @@
+import { CompactedNameOptionalUrl, CompactedNameUrl } from 'lib/rdf/types';
+
+type OrArray<T> = T | T[];
+
+export type BaseValue =
+  | CompactedNameOptionalUrl
+  | CompactedNameUrl
+  | string
+  | number;
```

Comment:
these are the base types that I chose to support for now (+ arrays of these base types)
do you think we should support others?

Reply:
(dates are part of `string`s)